libmscore/scale.h

```diff
@@ -0,0 +1,50 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2002-2012 Werner Schweer
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __SCALE_H__
+#define __SCALE_H__
+
+/**
+ \file
+ Definition of Scale class.
+ */
+
+#include "note.h"
+
+namespace Ms {
+
+class Scale {
+public:
+      bool loadScale(const QString&);
+      double getTuning(const Note* note) const;
+
+private:
+      QString getNextLine(QTextStream& in);
+
+      std::vector<double> originalNotes;
+      std::map<Tpc, double> computedNotes;
+
```

Comment:
I guess that's true. But the difference is so small that I don't mind storing double.

Would you mind adding your editor to this class, so we can change it? (we should change only the original notes). Or, you could send me the code and I'll integrate it.